tests/test_002_CPCandOptimizer.py

```diff
@@ -1472,7 +1469,7 @@ def test_margp_optimizer():
     assert set(r.tokens_t) == {'USDC', 'USDT'}
     assert r.errormsg is None
     assert r.is_error == False
-    assert r.time > 0
+    assert r.time >= 0
```

Comment:
This looks like a functional (behavioral) change; Are you sure that this is the right thing to do?
Even if yes, are you sure that it should be done in this PR?

Reply:
Sorry, that's a test file, so probably no issue (leaving the previous comment in place, just in case it IS an issue).